frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,168 @@
+    );
+    const pods = podList.items;
+    let pod = getPodData(pods, POD_NAME_PATTERNS.OCS);
+    let phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    pod = getPodData(pods, POD_NAME_PATTERNS.ROOK);
+    phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+    pod = getPodData(pods, POD_NAME_PATTERNS.NOOBA_OPERATOR);
+    phase = getPodPhase(pod);
+    expect(phase).toBe('Running');
+  });
+});
+
+describe('Test creation of Storage Cluster', () => {
+  it(
+    'creates a storage cluster',
+    async () => {
+      const nodes = await Installer.createStorageCluster();
+      const text = await crudView.resourceTitle.getText();
+      expect(text).toEqual(STORAGE_CLUSTER_NAME);
+      // Wait for cluster to come to ready state
+      await checkIfClusterIsReady();
+      // Verify all the nodes have the required labels
+      nodes.forEach((node) => {
+        expect(verifyNodeLabels(node, OCS_NODE_LABEL)).toBe(true);
+      });
+    },
+    15 * MINUTE,
+  );
+});
+
+describe('Tests for pods and storage classes', () => {
+  let pods = null;
+
+  beforeAll(async () => {
+    const podList = await JSON.parse(
+      execSync('kubectl get po -n openshift-storage -o json').toString('utf-8'),
+    );
+    pods = podList.items;
+  });
+
+  it('tests if ocs operator pods is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.OCS);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-operator is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.ROOK);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if nooba-operator is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.NOOBA_OPERATOR);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if nooba-core is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.NOOBAA_CORE);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mon-a is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.ROOK_CEPH_MON_A);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mon-b is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.ROOK_CEPH_MON_B);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mon-c is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.ROOK_CEPH_MON_C);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mgr is running', async () => {
+    const pod = getPodData(pods, POD_NAME_PATTERNS.ROOK_CEPH_MGR);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if 5 csi-cephfsplugin are running', async () => {
+    const podList = getPodData(pods, POD_NAME_PATTERNS.CSI_CEPHFS);
+    expect(podList.length).toBe(5);
+    podList.forEach((pod) => {
+      testPodIsRunning(getPodPhase(pod));
+    });
+  });
```

Comment:
These numbers are dependent on the number of Nodes. Can we do anything to verify accordingly?

Reply:
we can take it in a refactor or perhaps could you help me out and write that small change on top of it. This currently works for the standard setup.

Reply:
Can you just add a comment mentioning that it's 3 plugin + 2 provisioner pods? That will help everyone understand.

Reply:
ack